Extension/Settings.cs

```diff
@@ -1,13 +1,17 @@
-﻿using System.Xml.Serialization;
-using ModLib;
+﻿using ModLib;
 using ModLib.Attributes;
+using System.Collections.Generic;
+using System.Xml.Serialization;
 using YAPO.Global;
+using YAPO.MultipathUpgrade;
+using YAPO.MultipathUpgrade.Enum;
+using YAPO.MultipathUpgrade.Model;
 
 namespace YAPO
 {
     public class Settings : SettingsBase
     {
-        public const string InstanceId = Strings.MODULE_FOLDER_NAME + "Settings";
```

Comment:
This is the InstanceId for our settings class. So it should keep its Settings qualifier

Reply:
ok